crates/networking/p2p/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ream-p2p"
-authors.workspace = true
+authors = { workspace = true }
```

Comment:
please switch this back, you can update your local version of cargo sort to fix the error you seen

Reply:
this feedback wasn't addressed, but resolved was clicked, please switch it back to `authors.workspace`